README.md

```diff
@@ -127,6 +127,15 @@ We add `first_pass_output` to the rendering context, which is useful when you ne
 - `first_pass_output.source` contains the complete output of a first pass generation of the document.
 - `first_pass_output.lines` contains the same output as list of lines.
 
+### Jinja Filters
+
+The markdown files support three jinja filters within documents:
```

Comment:
Since the `rdm render` subcommand can be run on any type of text file (e.g., latex files too), I think it may be more precise to say:

"The `rdm render` subcommand provide a few extra filters to the Jinja context:"

Note that I remove the number "three" since I don't think it adds much value and it will make the documentation more likely to go stale if, e.g., someone adds more filters to the list without noticing they need to update the count.